src/packages/@ncigdc/modern_components/GeneExpression/inchlib/index.js

```diff
@@ -0,0 +1,4431 @@
+/* eslint-disable */
```

Comment:
Should we consider extracting our modified version of InCHlib to OncoJS?

That may also allow us not having to import a second version of jQuery into the app, and allow others to use the modified version.

Reply:
not sure if that's OK from a licensing standpoint but we could look into it

I'm considering removing jQuery altogether because Konva uses JavaScript selectors and it confuses things a bit.

Reply:
That makes total sense... as long as we keep the credits right and clear, and we respect the license terms, we should be able to generate a valid fork others can benefit from.

Wanna talk about it with the team when the time comes? 👍

Reply:
sounds good!